bin/git/git-node-help

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+echo "
+1) To land single a commit:
+   git node just <PRID>
```

Comment:
@joyeecheung just want to know am i mistaken or this should be `git-node` instead of `git node`
as we have `git-node` in `package.json`?

Reply:
The git custom commands work like this: if you have an executable named "git-*" on your path, the it will be run when you do "git *"